NMS-14526: Karaf commands for Pollerd/Collectd scheduler (#6105)
OpenNMS/opennms: docs/modules/operation/pages/deep-dive/service-assurance/shell/pollerd-schedule.adoc

[[ga-service-assurance-schedule-shell]]
= Pollerd Schedule

The `opennms:pollerd-schedule` Karaf Shell command can be used to output details of the Pollerd scheduler.

* List all collected statistics of the Pollerd daemon.
+
[source, karaf]
----
opennms:pollerd-schedule
----

> * Use `--services/-s` to output a number of scheduled services. The option's argument specifies the limit. Use `0` to display all scheduled services.

Reviewer: ```suggestion
* Use `--services/-s` to output a number of scheduled services.
The option's argument specifies the limit.
Use `0` to display all scheduled services.
```

+
[source, karaf]
----
opennms:pollerd-schedule --services 0
----

> * You can alter the output by specifying `--columns/-c` using a comma-seperated list of the columns to be displayed.

Reviewer: ```suggestion
* You can alter the output by specifying `--columns/-c` using a comma-seperated list of the columns to display.
```

+
[source, karaf]
----
opennms:pollerd-schedule --services 0 --columns nodeId,readyTimeMs
----

> * You can alter the order of the displayed service schedules by using the option `--option/-o`. To switch to a descending order specify the additional option `--desc/-d`.

Reviewer: ```suggestion
* You can alter the order of the displayed service schedules by using the option `--option/-o`.
To switch to a descending order specify the additional option `--desc/-d`.
```
